src/applications/vre/28-8832/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "appName": "Apply for Personalized Career Planning and Guidance with VA Form 28-8832",
+  "appName": "Apply for Personalized Career Planning and Guidance with VA Form 25-8832",
   "entryFile": "./app-entry.jsx",
   "entryName": "28-8832-planning-and-career-guidance",
   "rootUrl": "/careers-employment/education-and-career-counseling/apply-career-guidance-form-28-8832",
```

Comment:
If you ever need to update this path or entryName, it might be better to duplicate this entry with the new `entryName` & `rootUrl` first, then get devops to add a redirect to the new path. Then, remove this older entry.